tests/integration/cartography/intel/aws/test_cloudfront.py

```diff
@@ -0,0 +1,30 @@
+import cartography.intel.aws.cloudfront
+import tests.data.aws.cloudfront
+
+TEST_ACCOUNT_ID = '000000000000'
+TEST_REGION = 'eu-west-1'
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_repository_associations(neo4j_session):
+    data = tests.data.aws.cloudfront.GET_LIST_DISTRIBUTION
+    cartography.intel.aws.cloudfront.load_cloudfront_distributions(
+        neo4j_session,
+        data,
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+
+    expected_nodes = {
+        "arn:aws:cloudfront::000000000000:distribution/test1",
+        "arn:aws:cloudfront::000000000000:distribution/test2",
+    }
+
+    nodes = neo4j_session.run(
+        """
+        MATCH (r:CloudFrontDistribution) RETURN r.arn;
```

Comment:
Modify the test to also check for the Distribution Origins and the HAS_ORIGIN relationship.